src/main/java/ui/listpanel/ListPanel.java

```diff
@@ -212,10 +212,17 @@ private void setupKeyboardShortcuts() {
             if (NEW_COMMENT.match(event)) {
                 if (KeyPress.isValidKeyCombination(GOTO_MODIFIER.getCode(), event.getCode())) {
                     ui.getBrowserComponent().switchToTab(DISCUSSION_TAB);
-                } else if (ui.getBrowserComponent().isCurrentUrlIssue()) {
-                    ui.getBrowserComponent().switchToTab(DISCUSSION_TAB);
+                } else {
+                    if (!ui.getBrowserComponent().isCurrentUrlPrDiscussion()) {
+                        ui.getBrowserComponent().switchToTab(DISCUSSION_TAB);
+                    }
+
+                    while (!ui.getBrowserComponent().isCurrentUrlPrDiscussion()) {
+                        // wait
+                    }
```

Comment:
@acruis I would like to wait until a url is loaded in browser. Is there any method to do so?